vspace/src/main/webapp/WEB-INF/views/staff/spaces/spacelist.jsp

```diff
@@ -13,14 +20,54 @@
 			href="<c:url value="/staff/space/${space.id}" />"> <span
 				data-feather="box"></span> ${space.name}
 		</a> (Created on <span class="date">${space.creationDate}</span> by
-			${space.createdBy}) <a href="#" data-record-id="${space.id}"
-			data-url="<c:url value="/staff/space/${space.id}?${_csrf.parameterName}=${_csrf.token}"/>"
-			data-call-on-success="<c:url value="/staff/space/list"/>"
-			data-call-on-error="<c:url value="/staff/space/list"/>"
-			data-toggle="modal" data-target="#confirm-delete"><span
-				data-feather="trash-2"></span></a></li>
+			${space.createdBy}) 
+           <a href="javascript:checkSpaceLinkPresent('${space.id}', '<c:url value="/staff/" />', '?${_csrf.parameterName}=${_csrf.token}',$('#headerSpaceValue'))" class="checkSpaceLinkPresent" >
+               <span class="float-right checkSpaceLinkPresent" id="deleteSpace" data-feather="trash-2"></span>
+           </a>
+        </li>
 	</c:forEach>
 </ul>
-<jsp:include page="../../deleteModal.jsp">
-	<jsp:param name="elementType" value="Space" />
-</jsp:include>
\ No newline at end of file
+
+<!--  Ashmi changes start -->
+
+<div class="modal fade" id="confirm-space-delete" tabindex="-1" role="dialog"
+    aria-labelledby="myModalLabel" aria-hidden="true">
+    <div class="modal-dialog">
+        <div class="modal-content">
+            <div class="modal-header" data-spaceValue="" id="headerSpaceValue">
+                <h5 class="modal-title" id="deleteModalTitle">
+                    Confirm Deletion?
+                </h5>
+                <button type="button" class="close" data-dismiss="modal"
+                    aria-hidden="true">×</button>
+            </div>
+            <div class="modal-body">
+                <p id = "spaceData">
+                    Are you sure you want to delete ?
+                </p>
+                <div id="warningMessage">
+                <div class="text-danger">
+                Other spaces have links to this space!
+                </div>
+                </div>
+                <div id="exhibitionMessage">
+                <div class="text-danger">
+                This space is the exhibition start space! This will set exhibition to null.
```

Comment:
I think I made a comment about this warning message in the previous pull request. Also, I've changed the font size for the messages here, but it also needs to be changed on the space.jsp page.